src/boot_O2/__osMalloc.c

```diff
@@ -124,13 +124,14 @@ u8 __osMallocIsInitalized(Arena* arena) {
 
 /**
  * @brief Allocates at least \p size bytes of memory using the given \p arena.
- * The block of memory will be allocated from the beginning of the \p arena.
+ * The block of memory will be allocated from the beginning of \p arena.
  *
- * If there's not enough space in the given \p arena, this function will fail, returning `NULL`.
- * If \p size is zero, then an empty region of memory is returned. This region of memory must be freed to avoid memory
- * leaks.
+ *  - If there's not enough space in the given \p arena, this function will fail, returning `NULL`.
+ *  - If \p size is zero, then an empty region of memory is returned.
+ *  - This region of memory must be freed to avoid memory leaks.
  *
- * To avoid memory leaks, the returned pointer should be eventually deallocated using `__osFree` or `__osRealloc`.
+ * To avoid memory leaks, the returned pointer should be eventually deallocated using either `__osFree` or
```

Comment:
Basically this is the same thing, right?

Reply:
Yes...